backend/src/packages/chaiNNer_standard/image_adjustment/adjustments/stretch_contrast.py

```diff
@@ -0,0 +1,112 @@
+from __future__ import annotations
+
+from enum import Enum
+from typing import List, Tuple
+
+import numpy as np
+
+from nodes.groups import if_enum_group
+from nodes.properties.inputs import BoolInput, EnumInput, ImageInput, SliderInput
+from nodes.properties.outputs import ImageOutput
+from nodes.utils.utils import get_h_w_c
+
+from .. import adjustments_group
+
+
+def _stretch(img: np.ndarray, range_min: float, range_max: float) -> np.ndarray:
+    if range_min > range_max:
+        raise ValueError("min must be less than max")
+    if range_min == range_max:
+        return img * 0
+
+    return (img - range_min) / range_max
+
+
+class StretchMode(Enum):
+    AUTOMATIC = 0
+    PERCENTILE = 1
+    MANUAL = 2
+
+
+@adjustments_group.register(
+    schema_id="chainner:image:stretch_contrast",
+    description="Automatically stretches the histogram values in the given image.",
```

Comment:
You should mention here that it is similar to auto levels, since I'm sure that's something someone would search for

Reply:
Done.